improvelib/initializer/cli_params_def.py

```diff
@@ -279,23 +279,29 @@
      "default": ".pt", # Note! this default assumes PyTorch model
      # "required": True, # TODO if this is required param, then remove default
      "help": "[Dep?] File format to save the trained model."
-     },
-     {"name": "loss",  
+    },
+    {"name": "loss",  
      "type": str,
      "default": "mse",
      "help": "[Dep?] Loss metric."
-     },
+    },
     # ---------------------------------------
     {"name": "input_data_dir",
      "type": str,
      "action": StoreIfPresent,
      "help": "Dir where data for inference is stored."
-     },
+    },
     {"name": "input_model_dir",
      "type": str,
      "action": StoreIfPresent,
      "help": "Dir where model is stored.",
-     },
+    },
+    {"name": "calc_infer_scores",
```

Comment:
I am unsure what you are trying to achieve with the action StoreIfPresent. 
Isn't it clearer if the default is just False? Otherwise, what behaviour do you envision for:
- absent
- true
- false

Reply:
use `argparse.suppress` instead?

Reply:
@adpartin changed type to bool 
resolving conversation

Reply:
> use `argparse.suppress` instead?

@wilke I think you're right. We can achieve this with argparse.SUPPRESS. I think we should suppress `default` and `help` in all parameters that we plan to deprecate in this release. E.g., 

`
{"name": "train_ml_data_dir",
"default": argparse.SUPPRESS,
"help": argparse.SUPPRESS
}
` 

What's your opinion?